common/ai_army/10_AI_forts.txt

```diff
@@ -0,0 +1,19 @@
+province = {
+    war = {
+        active = {
+            is_at_war = yes
+        }
+        eval_multiply = {
+            factor = 2
+			AND = {
+				owned_by = FROM
```

Comment:
Is there any documentation on how this file is scoped?

Reply:
I've made it from example in vanilla folder

Reply:
Are we sure that FROM scope is the enemy country?

Reply:
I think FROM is the owner in this case.

Reply:
Is there any way confirm this?

Reply:
So here is developer comment
> # Scripts in ai_army can be used to influence how the army AI behaves when certain conditions (Defined in the "active" trigger) are met.
# It is intended not to control daily decision making of armies but to define a general attitude that will influence (or overwrite) hard-coded AI behaviour.
# The "active" trigger (I.e. the trigger that decides what attitude is currently active) is evaluated when any of the following happens:
#  * A war begins or ends
#  * A country gains or loses a province
#  * A year without wars or province changes passes

Reply:
My question is mostly ROOT vs FROM here.

Reply:
Well the comments puts "Owner of the army" out of the question. I think it mainly increases  the value of the province, in a scoring system AI use.

Reply:
Isn't "owned_by = FROM" redundant then if FROM is owner of province?

Reply:
But they province in question can be owned by the enemy, this is a purely defensive thing.